microsite/data/plugins/scaffolder-backend-dotnet.yaml

```diff
@@ -0,0 +1,9 @@
+---
+title: Scaffolder Backend .NET
```

Comment:
What do you think about calling it `Scaffolder .NET Actions`? Just to provide a bit more context

Reply:
Hi! What do you think about this suggestion?